vhost-dashboard.conf

```diff
@@ -0,0 +1,21 @@
+NameVirtualHost MY_HOST:80
```

Comment:
You have the explanation of this in the README
FYI: I use the NameVirtualHost close because I run multiple website on the same webserver.